docs/hub/billing.md

```diff
@@ -1,30 +1,75 @@
+We partner with cloud providers, like [AWS](https://huggingface.co/blog/aws-partnership), [Azure](https://huggingface.co/blog/hugging-face-endpoints-on-azure) and [Google Cloud](https://huggingface.co/blog/llama31-on-vertex-ai) to make it easy for customers to use Hugging Face directly in their cloud of choice. These solutions and usage are billed directly by the cloud provider. Ultimately we want people to be able to have great options to use Hugging Face wherever they build Machine Learning.
 
-- [Spaces](./spaces),
-- [AutoTrain](https://huggingface.co/docs/autotrain/index),
-- [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index)
-- [PRO subscription](https://huggingface.co/pricing) (for users)
-- [Enterprise Hub subscription](https://huggingface.co/enterprise) (for organizations)
+## Compute Services on the Hub
+
+We also directly provide compute services with [Spaces](./spaces), [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index) and the [Serverless Inference API](https://huggingface.co/docs/api-inference/index).
+
+While most of our compute services have a comprehensive free-tier, users and organizations can pay to access more powerful hardware accelerators.
+
+The billing for our compute services is usage-based, meaning you only pay for what you use. You can monitor your usage at any time from your billing dashboard, located in the settings menu of your user or organization.
+
+<div class="flex justify-center">
+	<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/billing-dashboard-light.png"/>
+	<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/billing-dashboard-dark.png"/>
+</div>
+
+### Available payment methods
+
+Hugging Face uses [Stripe](https://stripe.com) to securely process your payment information.
+
+The only payment method supported for Hugging Face compute services is credit cards.
+You can add a credit card to your account from your billing settings.
+
+<div class="flex justify-center">
+	<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/payment-method-light.png"/>
+	<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/payment-method-dark.png"/>
+</div>
+
+You also have an option to link your Hugging Face organization to your AWS account via [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-n6vsyhdjkfng2). Hugging Face compute service usage will then be included in your AWS bill.
+
+Read more in our [blog post](https://huggingface.co/blog/aws-marketplace).
+
+### Billing thresholds & Invoicing
```

Comment:
Does this allow setting up usage limits? E.g. not wanting to spend more than $Xk a month

Reply:
We do have such feature but it's not publicly available yet / there is no UI for it